cypress/support/cypress-support/src/utils.js

```diff
@@ -741,6 +741,72 @@ class FireLog {
 const fireLog = new FireLog();
 global.fireLog = fireLog;
 
+/**
+ * @module utils
+ * @globalfunction resolveAtRuntime
+ * @description Resolve the device variable from the preprocessed data for the given key
+ * @example
+ * resolveAtRuntime(["result.{{attribute}}", "result.styles.{{attribute}}"])
+ * resolveAtRuntime("manage_closedcaptions.set{{attribute.uppercaseFirstChar}}")
+ * resolveAtRuntime("value")
+ *
+ * @returns
+ * ['result.fontSize', 'result.styles.fontSize']
+ * "manage_closedcaptions.setFontSize"
+ * "1.5"
```

Comment:
I think the code probably handles this just fine but if the value is set to `1.5`, it should return as a number not as a String

Reply:
it is returning the value present in rutime env as-is, this i missed to remove quote